### ISSUE TITLE
Option create_rgba_format for image.load_buffer

### ISSUE THREAD

**Issue by HeliumVola** [defold/defold#9978]
I like to load JPG and PNG files from the web and combine them in ONE RGBA atlas/texture at runtime.

On Android I cannot use set_texture to copy the RGB buffer (from a loaded JPG) to the RGBA buffer created at runtime. It might be a restriction of OpenGL or a problem of an old OpenGL version. Works in the Engine and Windows, but not on my old Android 9 tablet.
Both buffers need to be in the same format, at least on my Android.

A solution would be to give an option create_rgba_format for image.load_buffer.
If create_rgba_format = true, image.load_buffer creates a RGBA buffer for a loaded JPG

**Comment by britzl** [comment]
> On Android I cannot use set_texture to copy the RGB buffer (from a loaded JPG) to the RGBA buffer created at runtime. It might be a restriction of OpenGL or a problem of an old OpenGL version. Works in the Engine and Windows, but not on my old Android 9 tablet.

Are you getting an error when you try this?

**Comment by HeliumVola** [comment]
Yes Error 1282 GL_INVALID_OPERATION in graphics_opengl.cpp. Line 4043 I think. Defold 1.9.6

**Comment by britzl** [comment]
@HeliumVola could you please share a small project where this can be reproduced?

**Comment by HeliumVola** [comment]
[loadresource.zip](https://github.com/user-attachments/files/18350574/loadresource.zip)

Works in editor, crash on my Huawei/Android 9
